Document how to make a feature request

ivelasq opened the pull request:
Hello! @lgibson7 and I are submitting this PR for https://github.com/r-devel/rdevguide/issues/134, also https://github.com/r-devel/r-dev-day/issues/57.

Thanks for considering!

llrs commented on 05-submitting_feature_requests.Rmd:
```diff
@@ -0,0 +1,22 @@
+
+![The "Wishlist" component in Bugzilla's bug submission screen.](img/feature-requests.png)
+
+Similar to bugs, you should ensure that the feature request is [not already reported](https://contributor.r-project.org/rdevguide/ReportingBugs.html#checking-if-a-bug-is-already-reported) and [follow best practices](https://contributor.r-project.org/rdevguide/ReportingBugs.html#good-practices-in-reporting-bugs-expectations-of-a-good-bug-report) whenever possible.
```
Please use the section to link instead of the full link (in case the section is moved somewhere else) like you did for the 03-issue_tracking.Rmd

llrs reviewed (commented):
Looks good! Many thanks for submitting this PR! 

I see some file additions like the rdevguide.log or the render*.rds file. Are they needed? Great call on removing the rdevguide.tex file.

ivelasq commented:
Thanks, @llrs! I have edited the links. I don't know where the render.rds or rdevguide.log files came from, they must have come from building the book. I removed them and added rdevguide.log to the .gitignore file.

llrs commented:
Thanks for adding the file to .gitignore! Many thanks both for your PR and contributions!! I think a bot will add your contributions to the book !! Let me know if it doesn't after I merge this.

SaranjeetKaur commented:
@all-contributors
please add @ivelasq for content, doc, ideas.
